internal/services/ordinals.go

```diff
@@ -46,6 +47,14 @@ func (s *Services) verifyViaOrdinalService(
 	}
 
 	for index, output := range outputs {
+		// Check the order of the response is the same as the request
+		if output.Transaction != utxos[index].Txid {
+			return nil, types.NewErrorWithMsg(
```

Comment:
Do we know why this might happen? Is it possible that we could sort the output to match the utxos?